### DISCUSSION TITLE
Use of TracePipeline from agent and collector

### DISCUSSION THREAD

**Discussion by ajprakas** [open-telemetry/opentelemetry-java-instrumentation#2045] [Q&A]
I am using this example- https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples/demo to instantiate a open telemetry agent and collector and publish spans to jaeger and zipkin backends via agent.How do I know which are the ports jaeger and zipkin receivers of agent are listening at so that I publish span to those port from my app.

after starting collector from docker-compose , I see agent, collector , jaeger and zipkin continers are running.

![image](https://user-images.githubusercontent.com/63052413/104553550-da0bfd00-5660-11eb-8a08-3674c8c3318d.png)

**Comment by trask** [comment]
hi @ajprakas, it looks like this is a duplicate of your post here? https://github.com/open-telemetry/opentelemetry-java/discussions/2523
